<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="FullImageView" name="fullImageViewWidget" native="true"/>
      <widget class="SelectionDetailsView" name="selectionDetailsView" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1209</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Directory"/>
    <addaction name="actionSelect_Image"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSelection">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionClear_Current_Selection"/>
    <addaction name="actionClear"/>
    <addaction name="actionNext_Image"/>
    <addaction name="actionPrev_Image"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSelection"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear_Current_Selection"/>
   <addaction name="actionClear"/>
   <addaction name="separator"/>
   <addaction name="actionPrev_Image"/>
   <addaction name="actionNext_Image"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Directory">
   <property name="text">
    <string>Open Directory</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear All Selections</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
  <action name="actionNext_Image">
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionPrev_Image">
   <property name="text">
    <string>Prev Image</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionSelect_Image">
   <property name="text">
    <string>Select Image</string>
   </property>
  </action>
  <action name="actionClear_Current_Selection">
   <property name="text">
    <string>Clear Current Selection</string>
   </property>
   <property name="toolTip">
    <string>Clear Current Selection</string>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FullImageView</class>
   <extends>QWidget</extends>
   <header>fullimageview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectionDetailsView</class>
   <extends>QWidget</extends>
   <header>selectiondetailsview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
